src/main.js

```diff
@@ -111,11 +143,19 @@ function showUsersIdeaCard() {
   if (savedIdeas.length) {
     ideaGallery.innerHTML = "";
     for (var i = 0; i < savedIdeas.length; i++) {
+
+      if (savedIdeas[i].star === false) {
+        var visibleStar = "assets/star.svg";
+        var starClassName = "star-inactive";
+      } else if (savedIdeas[i].star === true){
+        visibleStar = "assets/star-active.svg";
+        starClassName = "star-active";
+      }
+
       var ideaCardTemplate =
       `<section class="box" id="${savedIdeas[i].id}">
         <section class="card-top">
-          <input type="image" src="assets/star.svg" name="star-inactive" class="star-inactive" id="star-inactive" />
-          <input type="image" src="assets/star-active.svg" name="star-active" class="star-active hide" id="star-active" />
+          <input type="image" src=${visibleStar} name="star" class=${starClassName} id="star" />
```

Comment:
I took away the hide class to hide the active + inactive star. Instead, there are 2 options in the if/else on line 147. If the class instance property star === false, the inactive star shows, if true the active one shows. The src + className values change here since they're what's being displayed + the class tag grabbed from the eventListener above.